tests/e2e/pageobjects/dashboard/GetStarted.ts

```diff
@@ -49,13 +49,6 @@ export class GetStarted {
         await this.driverHelper.waitAndClick(sampleLocator, timeout);
     }
 
-    async selectSample(sampleName: string, timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
```

Comment:
Could you explain why you decided to delete this method?

Reply:
This method does not make sense in the current situation - if you click on an item, the new workspace will be opened. So it fails when waiting for `waitSampleSelected`.